### ISSUE TITLE
Should be able to execute recorded script

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#597]
## Expected behavior
Should be able to execute recorded script

## Actual behavior
Getting error
```
SyntaxError: Identifier 'below' has already been declared
    at new Script (vm.js:80:7)
    at createScript (vm.js:274:10)
    at Object.runInThisContext (vm.js:326:10)
    at Module._compile (internal/modules/cjs/loader.js:664:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
```
**Notes**
The require has repeated imports
```
const { openBrowser, goto, below, textBox, into, write, press, below, textBox, into, write, press, below, click, click, button, click, click, click, click, checkBox, click, click, closeBrowser } = require('taiko');
                                                               ^
```
**Steps to replicate**
In the REPL
```
> openBrowser();
 ✔ Browser opened
> goto("cleartrip.com");
 ✔ Navigated to url http://cleartrip.com
> write("Bangalore, IN - Kempegowda International Airport (BLR)",into(textBox(below("From"))));
 ✔ Wrote Bangalore, IN - Kempegowda International Airport (BLR) into the text field Below From
> press("Enter")
 ✔ Pressed the Enter key
> write("New Delhi, IN - Indira Gandhi Airport (DEL)",into(textBox(below("To"))));
 ✔ Wrote New Delhi, IN - Indira Gandhi Airport (DEL) into the text field Below To
> press("Enter")
 ✔ Pressed the Enter key
> click("24",below("June 2019"))
 ✔ Clicked element matching text "24"
> click("Search flights");
 ✔ Clicked element matching text "Search flights"
> click(button("Search flights"));
 ✔ Clicked button with label Search flights 
> click("Duration")
 ✘ Error: Element with text Duration not found, run `.trace` for more info.
> click(button("Search flights"))
 ✔ Clicked button with label Search flights 
> click(button("Search flights"))
 ✔ Clicked button with label Search flights 
> click(button("Book"))
 ✔ Clicked button with label Book 
> .code
```
This generates 
```
const { openBrowser, goto, below, textBox, into, write, press, below, textBox, into, write, press, below, click, click, button, click, button, click, button, click, button, click, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto("cleartrip.com");
        await write("Bangalore, IN - Kempegowda International Airport (BLR)",into(textBox(below("From"))));
        await press("Enter");
        await write("New Delhi, IN - Indira Gandhi Airport (DEL)",into(textBox(below("To"))));
        await press("Enter");
        await click("24",below("June 2019"));
        await click("Search flights");
        await click(button("Search flights"));
        await click(button("Search flights"));
        await click(button("Search flights"));
        await click(button("Book"));
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```
> There are repeated import elements. Thus this script cannot be executed

**Version**
```
d4a99381515a3a6262f829eee79a8e00197ada05
```

**Comment by BugDiver** [comment]
This should be fixed  in  c2b7de7

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the commit version ```c2b7de7 ```. Kindly find the below updated code generated  for the above scenario
```
const { openBrowser, overridePermissions, goto, below, textBox, into, write, press, click, text, button, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await overridePermissions('https://www.cleartrip.com', ['notifications']);
        await goto("https://www.cleartrip.com");
        await write("Bangalore, IN - Kempegowda International Airport (BLR)",into(textBox(below("From"))));
        await press("Enter");
        await write("New Delhi, IN - Indira Gandhi Airport (DEL)",into(textBox(below("To"))));
        await press("Enter");
        await click("Depart on");
        await click("24",below("June 2019"));
        await click("Search flights");
        await text("Search flights").exists();
        await click("Search flights");
        await press("Tab");
        await click("Search flights");
        await click(button({id: 'SearchBtn'}));
        await click("Duration");
        await click("Book");
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```
